package.json

```diff
@@ -289,7 +288,7 @@
         "react-responsive": "8.2.0",
         "react-router-dom": "^5.0.0",
         "react-scrollbars-custom": "^4.0.21",
-        "react-styleguidist": "^8.0.6",
+        "react-styleguidist": "12.0.1",
```

Comment:
do we want to keep the caret?

Reply:
I think we should just freeze it, we dont gain any benefit for letting the minor or fix version in when we plan to deprecate for real this time